datasource/ethereum/src/ethereum_adapter.rs

```diff
@@ -1084,4 +971,209 @@ where
             .and_then(move |output| call.function.decode_output(&output).map_err(From::from)),
         )
     }
+
+    fn triggers_in_block(
+        self: Arc<Self>,
+        logger: Logger,
+        chain_store: Arc<dyn ChainStore>,
+        subgraph_metrics: Arc<SubgraphEthRpcMetrics>,
+        log_filter: EthereumLogFilter,
+        call_filter: EthereumCallFilter,
+        block_filter: EthereumBlockFilter,
+        ethereum_block: BlockFinality,
+    ) -> Box<dyn Future<Item = EthereumBlockWithTriggers, Error = Error> + Send> {
+        Box::new(match &ethereum_block {
+            BlockFinality::Final(block) => Box::new(
+                self.blocks_with_triggers(
+                    logger,
+                    chain_store,
+                    subgraph_metrics,
+                    block.number(),
+                    block.number(),
+                    log_filter.clone(),
+                    call_filter.clone(),
+                    block_filter.clone(),
+                )
+                .map(|blocks| {
+                    assert!(blocks.len() <= 1);
+                    blocks
+                        .into_iter()
+                        .next()
+                        .unwrap_or(EthereumBlockWithTriggers::new(vec![], ethereum_block))
+                }),
+            )
+                as Box<dyn Future<Item = _, Error = _> + Send>,
+            BlockFinality::NonFinal(fat_block) => Box::new(future::ok({
```

Comment:
Could we rename `fat_block` to `full_block`, since that's more in line with e.g. `load_full_block` etc.

Reply:
Done, thanks